package.json

```diff
@@ -57,7 +57,7 @@
     "dependency-tree": "^6.2.1",
     "husky": "0.14.3",
     "intern": "^4.3.1",
-    "lint-staged": "6.0.0",
+    "lint-staged": "^10.0.9",
```

Comment:
With v10 of lint-staged, can you remove the `git add` task from the lint-staged config as per [this](https://github.com/okonet/lint-staged#v10)